content/blog/how-elevate-supports-business-needs.md

```diff
@@ -0,0 +1,182 @@
+
+Upgrading is most of the time the less painfull way to get your customer to use a new OS,
+as with migrating to a new server there's always something that takes some debugging where in-place upgrades keep it mostly intact,
+which is important, because as a techie we tend to forget that customers mostly want the servers to just work to support their business needs.
+
+That still doesn't make up for the CentOS 6 servers currently around, which in all honesty should've been gone a long time ago already,
+in which as a MSP you can only do so much for your customer,
+if in the end they use an application for their business that for some reason only runs on CentOS 6,
+or their developer left a long time ago, well, good luck with replacing that server...
+
+It gives however a great reason to have some fun with the new release of [ELevate](https://almalinux.org/blog/2024-04-25-elevate-supports-centos-6-to-centos-7/), as it now supports ELevating CentOS 6 too!
+
+Nowadays there are less and less excuses to make, as [DirectAdmin](https://www.directadmin.com) for example made it even possible to run php 5.6 code on AlmaLinux 9,
+which is a great way of offering your customer a up-to-date operating system if they're really dependent on php 5.6, which gives us one less thing to worry about.
+
+So when CloudFest and AlmaLinux day came around great things happened at the hackathon and the idea for ELevating CentOS 6 servers to CentOS 7 was born, which is really exciting news.
+
+Once benny posted that ELevate was available for CentOS 6 I was really excited to try it on one of our servers, so after a bit of cloning and snapshotting off we went!
+
+## Practical tips from a real life ELevate user
+While after all the ELevate process itself is quite painless, it can be quite hard to get there.
+I really recommend following/reading the detailed [wiki](https://wiki.almalinux.org/elevate) for the different available migrations.
+
+ELevate is really good at identifying (potential) issues which you must resolve beforehand, below you can read some of those.
+
+### /boot
+One that bothered us the most was that the /boot partition is almost always in need of more space,
+while in more recent installations it was a matter of cleaning up old kernels, in our CentOS 6 servers that really wasn't happening as most servers had a 128Mb /boot partition.
+
+So I had to move the /boot partition to a new partition which was bigger, again, something to keep in mind,
+while ELevate is a great tool to help deprecating those last CentOS 6 servers, it still might be cleaner to do it from scratch some day.
+
+While moving the partition is quite an easy way, it can be quite dirty if you put it directly behind your data partition,
+you might want to consider moving that further back and just expanding the current /boot partition itself,
+or even simpler, move it to a seperate disk, for which the steps are as follows:
```

Comment:
```suggestion
While moving the partition is quite an easy way, it can be quite dirty if you put it directly behind your data partition. You might want to consider just expanding the current /boot partition itself, or even simpler, move it to a seperate disk, for which the steps are as follows:
```